worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/PlatformCommandManager.java

```diff
@@ -376,13 +388,14 @@ private void registerAlwaysInjectedValues() {
 
             CommandManager manager = commandManagerService.newCommandManager();
 
-            handlerInstance.accept((handler, instance) -> {
-                this.registration.register(
-                        manager,
-                        handler,
-                        instance
-                );
-            });
+            //FAWE start
+            handlerInstance.accept((handler, instance) ->
+                    this.registration.register(
+                    manager,
+                    handler,
+                    instance
```

Comment:
is this tabbing correct?

Reply:
As correct as our commented-out checkstyle plugin. At some point we should bother enforcing a style across all files. There are way more inconsistencies, but, there is no followed standard atm so I didn't bother with 4/8/how many tabs we use.